test/common/config.go

```diff
@@ -17,38 +17,43 @@ limitations under the License.
 package common
 
 import (
-	"github.com/knative/eventing/test/base"
+	"github.com/knative/eventing/test/base/resources"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 // DefaultClusterChannelProvisioner is the default ClusterChannelProvisioner we will run tests against.
-const DefaultClusterChannelProvisioner = base.InMemoryProvisioner
+const DefaultClusterChannelProvisioner = resources.InMemoryProvisioner
 
 // ValidProvisionersMap saves the provisioner-features mapping.
 // Each pair means the provisioner support the list of features.
 var ValidProvisionersMap = map[string]ChannelConfig{
-	base.InMemoryProvisioner: ChannelConfig{
-		Features: []Feature{FeatureBasic},
+	resources.InMemoryProvisioner: ChannelConfig{
+		Features:     []Feature{FeatureBasic},
+		CRDSupported: true,
 	},
-	base.GCPPubSubProvisioner: ChannelConfig{
+	resources.GCPPubSubProvisioner: ChannelConfig{
 		Features: []Feature{FeatureBasic, FeatureRedelivery, FeaturePersistence},
 	},
-	base.KafkaProvisioner: ChannelConfig{
+	resources.KafkaProvisioner: ChannelConfig{
 		Features:     []Feature{FeatureBasic, FeatureRedelivery, FeaturePersistence},
 		CRDSupported: true,
 	},
-	base.NatssProvisioner: ChannelConfig{
+	resources.NatssProvisioner: ChannelConfig{
 		Features: []Feature{FeatureBasic, FeatureRedelivery, FeaturePersistence},
 	},
 }
 
+// ChannelConfig includes general configuration for a Channel provisioner.
 type ChannelConfig struct {
 	Features     []Feature
 	CRDSupported bool
 }
 
+// OperatorChannelMap saves the mapping between provisioners and CRD channel typemeta.
+// TODO(Fredy-Z): this map will not be needed anymore when we delete the provisioner implementation.
 var OperatorChannelMap = map[string]*metav1.TypeMeta{
-	base.KafkaProvisioner: KafkaChannelTypeMeta,
+	resources.KafkaProvisioner:    KafkaChannelTypeMeta,
```

Comment:
sounds weird that in an operator map be have the keys to be Provisioner. 
Can we just rename to resources.Kafka ?
I'm fine if not

Reply:
I would prefer to keep them named as `provisoner`. Since we will remove `provionser` soon, I think it's not worth to change it and add extra confusion now.